lib/Event/EventOptionsTrait.php

```diff
@@ -19,14 +21,14 @@
     /**
      * This array is used as key value storage for the options.
      *
-     * @var array
+     * @var OptionsResolver
```

Comment:
The options resolver only "resolves" an array of options to according to a configuration. The options themselves are always an array.